### PR TITLE
Fix: qualify public table names for target PG by default schema in multiple schemas case

### PR DESCRIPTION
**Pull request by priyanshi-yb**

### CODE REVIEW

File: yb-voyager/src/tgtdb/postgres.go [hunk 1]
```diff
@@ -285,9 +285,21 @@ outer:
...
+		return schema, false
+	} else if slices.Contains(schemas, "public") {
+		return "public", false
+	} else {
+		return "", true
+	}
+}
+
 func (pg *TargetPostgreSQL) qualifyTableName(tableName string) string {
+	defaultSchema, _ := getDefaultPGSchema(pg.tconf.Schema)
```
Comment (amit-yb) [inline yb-voyager/src/tgtdb/postgres.go:300]: What about cases where there isn't a clear "default" schema?

File: yb-voyager/src/tgtdb/postgres.go [hunk 2]
```diff
@@ -285,9 +285,21 @@ outer:
 	return nil
 }
 
+func getDefaultPGSchema(schema string) (string, bool) {
```
Comment (amit-yb) [inline yb-voyager/src/tgtdb/postgres.go:288]: Add a comment stating the meaning of the `bool` return value?

**Review by amit-yb — APPROVED** [review]
LGTM

File: yb-voyager/src/tgtdb/postgres.go (hunk 1 above)
Comment (priyanshi-yb) [reply yb-voyager/src/tgtdb/postgres.go]: hmm, handled it now.

**Pull request merged**